src/glados/engine.py

```
@@ -23,7 +23,12 @@
 from .TTS import tts_glados, tts_kokoro
 from .utils import spoken_text_converter as stc
 
-logger.remove(0)
+try:
+    logger.remove(0)
+except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)